openapi.yaml

```diff
@@ -16,175 +16,374 @@ host: api.oregonstate.edu
+      responses:
+        '201':
+          description: Successfully create a game
+          schema:
+            $ref: '#/definitions/GameResult'
+          headers:
+            Location:
+              type: string
+              format: url
+              description: Location of the newly created game
+        '400':
+          description: Invalid game object
+          schema:
+            $ref: '#/definitions/Errors'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/Errors'
+    get:
+      summary: Info of a list of game
+      tags:
+        - games
+      operationId: getGames
+      parameters: 
         - $ref: '#/parameters/authorization'
-        - $ref: '#/parameters/pageNumber'
-        - $ref: '#/parameters/pageSize'
-        - in: query
-          name: species
-          description: Search by pet species
-          required: false
-          type: string
       responses:
         '200':
           description: Successful response
           schema:
-            $ref: '#/definitions/PetsResult'
+            $ref: '#/definitions/GameList'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/Errors'
+  /games/{gameId}:
+    get:
+      summary: Info for a specific game
+      tags:
+        - games
+      description: Get a game by its gameid
+      operationId: getGameByGameId
+      parameters: 
+        - $ref: '#/parameters/authorization'
+        - $ref: '#/parameters/gameId'
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/GameResult'
+        '404':
+          description: Game not found
+          schema:
+            $ref: '#/definitions/Errors'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/Errors'
+    patch:
+      tags:
+        - games
+      summary: Update a game specified by its id
+      operationId: patchGameByGameId
+      consumes:
+        - application/json
+      parameters: 
+        - $ref: '#/parameters/gameId'
+        - name: updateBody
+          in: body
+          description: Object that will overwrite the game specified by its id
+          required: true
+          schema:
+            $ref: '#/definitions/GameUpdateBody'
+      responses:
+        '200':
+          description: Successfully update a game
+          schema:
+            $ref: '#/definitions/GameResult'
         '400':
-          description: Bad request
+          description: Invalid game object
+          schema:
+            $ref: '#/definitions/Errors'
+        '404':
+          description: Game not found
           schema:
             $ref: '#/definitions/Errors'
         '500':
           description: Internal server error
           schema:
             $ref: '#/definitions/Errors'
-  /pets/{id}:
+    delete:
+      tags:
+      - games
```

Comment:
Indentation is missing.

Reply:
Like @tsoliangwu0130 mentioned, `- games` should be indented here to be consistent with the rest of the document